### PR TITLE
refs :#121 and fixes #121 SQL Day2 Task done

### PR DESCRIPTION
**Pull request by mastermind04**
SQL DAY 2 Assignment complete

### CODE REVIEW

File: Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+--TO EXECUTE THIS QUERY FIRST REMOVE UNIQUE CONSTRAINT FROM EMAIL ID
+UPDATE Employees SET Email = 'not available' WHERE DepartmentID = 80 AND CommissionPct BETWEEN 0 AND 0.20
+
+-----------------------------------ASSIGNMENT 5----------------------------------
+
+
+/*Write a SQL statement to change the Email column of employees table with �not available�
+for those employees who belongs to the �Accouning� department.*/
+
+-------Accounting  Department is not Found
```
Comment (radixdt12020) [inline Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql:170]: but you can write a query for the same

File: Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+SET Salary = CASE WHEN DepartmentId = 40 THEN Salary*1.25    
+		 WHEN DepartmentId = 90 THEN Salary*1.15
+		 WHEN DepartmentId = 110 THEN Salary*1.10
+		 ELSE Salary 
+		 END
+
+-----------------------------------ASSIGNMENT 9----------------------------------
+
+/*Write a SQL statement to increase the minimum and maximum salary of PU_CLERK by 
+2000 as well as the salary for those employees by 20% and commission by 10% .*/
```
Comment (radixdt12020) [inline Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql:206]: Why this Assignment is pending?

**Review by radixdt12020 — APPROVED** [review]

File: Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql (hunk 1 above)
Comment (mastermind04) [reply Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql:170]: okay i'll write it down

File: Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql (hunk 2 above)
Comment (mastermind04) [reply Modules/SQL/Day2/Smit vadhiya/SQL-Day2-Assignments.sql:206]: There is no minimum and maximun salary column

**Pull request merged**